tests/integration/src/test/java/org/netxms/utilities/TestHelper.java

```diff
@@ -156,5 +156,17 @@ public static void checkObjectCreated(NXCSession session) throws IOException, NX
 
       }
    }
+   public static AbstractObject findObject(NXCSession session, String objectName) throws IOException, NXCException
```

Comment:
NXCSession has method public AbstractObject findObjectByName(final String name) no need to reimplement it